yellowbrick/features/pcoords.py

```diff
@@ -333,34 +346,15 @@ def fit(self, X, y=None, **kwargs):
         self : instance
             Returns the instance of the transformer/visualizer
         """
+        # Determine the features, classes, and colors
+        super(ParallelCoordinates, self).fit(X, y)
```

Comment:
See my other comment about how/whether we could better document our use of calling `super(Visualizer, self).fit(X,y)` inside `fit` for different visualizers &mdash; not sure what would make most sense but I know that this would help me reason better about our class hierarchy as a contributor!